library/src/test/java/bio/terra/landingzone/service/landingzone/azure/LandingZoneServiceTest.java

```diff
@@ -507,6 +507,7 @@ private LandingZoneRecord createLandingZoneRecord() {
         "subscriptionId",
         "tenantId",
         billingProfileId,
+        null,
```

Comment:
I think I mentioned this on a previous iteration but: it'd be good to add or modify one of these tests to exercise storage of a region. These all appear to be storing `null`.